src/TwitchRequest.php

```diff
@@ -47,7 +52,7 @@ class TwitchRequest
      */
     protected function sendRequest($method, $endpoint, $params = [], $accessToken = null)
     {
-        $client = $this->getNewHttpClient($method, $params, $accessToken);
+        $client = $this->getNewHttpClient($method, $params, $endpoint, $accessToken);
         $response = $client->request($method, $endpoint);
         $responseBody = $response->getBody()->getContents();
```

Comment:
@nicklaw5 When refreshing a token, we're missing the client ID, apparently. The `$responseBody` here, when trying to refresh a token, is:

```
{"status":400,"message":"missing client id"}
```

Reply:
@echosa Thanks, I will take a closer look tomorrow.

Reply:
Appreciated. I debugged a bit and can't see why it's not picking up the client ID. It seems to be properly sent. Hopefully you'll have better luck.